Script/navBar.js

```diff
@@ -0,0 +1,31 @@
+let navBarActive = document.getElementById("navBarActive").innerHTML;
+
+if (navBarActive == "homePage")
+    {
+        document.getElementById("homeNav").classList.add("active")
+        console.log("homepage is open")
+    }
+    else if (navBarActive == "aboutPage")
+    {
+        document.getElementById("aboutNav").classList.add("active")
+        console.log("aboutPage is open")
+    }
+    else if (navBarActive == "projectPage")
+    {
+        document.getElementById("projectNav").classList.add("active")
+        console.log("projectPage is open")
+    }
+    else if (navBarActive == "loginPage")
+    {
+        document.getElementById("accountNav").classList.add("active")
+        console.log("accountPage is open")
+    }
+    else if (navBarActive == "registerPage")
+    {
+        document.getElementById("accountNav").classList.add("active")
+        console.log("accountPage is open")
+    }
+    else 
+    {
+        console.log("nothing is open")
+    }
```

Comment:
Remove console log. You use it when development. But when done you remove it. Otherwise could lead to huge security risks

Reply:
Test